analyzer-comments/python/general/no_return.md

```diff
@@ -0,0 +1,8 @@
+# general missing return statement
+
+A `return` statment is not being used in this function to return the result string, and this code should have failed one or more test cases during testing.
```

Comment:
Is it always a "result string"?

Reply:
ah!  good catch.  No.  Not for general feedback.  I was in my `two-fer` mode.  LOL.  Will note for future cleanup.  😄 .